### PR TITLE
Bump the minimum versions of JWTDecode.swift and SimpleKeychain (#777)

### PR DESCRIPTION
**Pull request by bdolewski-intellias**
- [ ] All new/changed/fixed functionality is covered by tests (or N/A)
- [ ] I have added documentation for all new/changed functionality (or N/A)

### CODE REVIEW

**Review by salmin-skelia — APPROVED** [review]

**Comment by salmin-skelia** [comment]
Why do we need to fork Auth0 at all?

**Comment by bdolewski-intellias** [comment]
> Why do we need to fork Auth0 at all?

It was a part of workaround for `Auth0` problem. I see no reason to keep it. 

@dvosscricut did you created this fork? Can we remove it?

File: Auth0/ClaimValidators.swift [hunk 1]
```diff
@@ -300,7 +300,7 @@ struct IDTokenOrgNameValidator: JWTValidator {
 
     func validate(_ jwt: JWT) -> Auth0Error? {
         guard let actualOrgName = jwt.claim(name: "org_name").string else { return ValidationError.missingOrgName }
-        guard actualOrgName.caseInsensitiveCompare(expectedOrgName) == .orderedSame else {
+        guard actualOrgName == expectedOrgName.lowercased() else {
```
Comment (salmin-skelia) [inline Auth0/ClaimValidators.swift:303]: I think that may be a bug.

`caseInsensitiveCompare` will compare both strings lowercased(), so I think you need to make `actualOrgName` lowercased() as well

**Review by salmin-skelia — APPROVED** [review]

File: Auth0/ClaimValidators.swift (hunk 1 above)
Comment (bdolewski-intellias) [reply Auth0/ClaimValidators.swift:303]: @salmin-skelia do we need this fork at all?

**Review by salmin-skelia — APPROVED** [review]

File: Auth0/ClaimValidators.swift (hunk 1 above)
Comment (salmin-skelia) [reply Auth0/ClaimValidators.swift:303]: I don't know

**Pull request closed without merging**